_posts/2022-12-01-predict-proba.md

```diff
@@ -0,0 +1,67 @@
+---
+#### Blog Post Template ####
+
+#### Post Information ####
+# title: "What we should and should not expect from `predict_proba`"
+title: "What to expect from `predict_proba`"
+date: December 1, 2022
+
+#### Post Category and Tags ####
+# Format in titlecase without dashes (Ex. "Open Source" instead of "open-source")
+categories:
+  - Updates
+tags:
+  - Machine Learning
+
+#### Featured Image ####
+featured-image: sorting.png
+
+#### Author Info ####
+# Can accomodate multiple authors
+# Add SQUARE Author Image to /assets/images/author_images/ folder
+postauthors:
+  - name: Alexandre Perez-Lebel
+    website: https://perez-lebel.com
+    email: alexandre.perez@inria.fr
+    image: alexandre_perez.jpeg
+usemathjax: true
+---
+<div>
+  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="" title="Image by storyset on Freepik">
+  {% include postauthor.html %}
+</div>
+
+In classification, many situations call for estimated probabilities beyond class labels. For example in decision making, cost sensitive learning or causal inference.
+These probability estimates are typically accessible from the `predict_proba` method of scikit-learn's classifiers.
+
+However, the quality of the estimated probabilities must be validated to provide trustworthiness, ensure fairness and robustness to operating conditions.
+To be reliable, the estimated probabilities must be close to the true underlying posterior probabilities of the classes `P(Y=1|X)`.
+
+Similarly to validating a discriminant classifier through accuracy or ROC curves, tools have been developed to evaluate a probabilistic classifier.
+Calibration is one of them [1-4]. Calibration is used as a proxy to evaluate the closeness of the estimated probabilities to the true ones. Many recalibration techniques have been developed to improve the estimated probabilities (see [scikit-learn's user guide on calibration](https://scikit-learn.org/stable/modules/calibration.html)). Estimated probabilities of a calibrated classifier can be interpreted as probability of correctness on population of same estimated probability, but not as the true posterior class probability.
+
+Indeed, it is important to highlight that calibration only captures part of the error on the estimated probabilities. The remaining term is the grouping loss [5]. Together, the calibration and grouping losses fully characterize the error on the estimated probabilities, the epistemic loss.
```

Comment:
You are too formal. Give us the intuitions of why calibration is not the full story, rather than the maths.